imports/plugins/core/graphql/server/schemas/cart.graphql

```diff
@@ -0,0 +1,346 @@
+# Cart Schema
+"The cart holds selected items until order is placed."
+type Cart implements Node {
+  "The Cart ID"
+  _id: ID!
+
+  # We're currently using User on the Cart.
+  # Enforce database uniqueness such that an account only has one cart.
+  "The account that owns the cart. Some carts are created for anonymus users. Anonymous carts have a null account."
+  account: Account
+
+  "The date and time at which the cart was created, which is when the first item was added to it."
+  createdAt: DateTime!
+
+  "The items that have been added to the cart. A cart is not created until the first item is added. Items can be removed from a cart, and a cart is not deleted if all items are removed from it. Because all items may have been removed, this may be an empty array."
+  items: [CartItem]!
+
+  "The shop that owns the cart."
+  shop: Shop!
+
+  # This value
+  "The date and time at which this cart was last updated."
+  updatedAt: DateTime!
+
+  # Read-only: Carts that don't have an account should be considered anonymous
+  # and should be cleaned up after some time. This should not be set explicitly
+  # on the cart but can be derived and add to the Cart as a read-only
+  # attribute (read, value from a function.) The cart expiration ttl can then be
+  # modified without having to modify cart records.
+  # This value should not be stored in the database.
+  # Cart expiry is: updated_at + cart_expiry_threshold < now
+  "The date and time at which the cart will expire."
+  expiresAt: DateTime!
+}
+
+"Wraps a list of `CartItem`s, providing pagination cursors and information."
+type CartItemConnection implements NodeConnection {
+  edges: [CartItemEdge]
+  nodes: [CartItem]
+  pageInfo: PageInfo!
+  totalCount: Int!
+}
+
+"A connection edge in which each node is a `CartItem` object"
+type CartItemEdge implements NodeEdge {
+  cursor: ConnectionCursor!
+  node: CartItem
+}
+
+"""
+A single item in a cart. The item contains information about an intended purchase.
+"""
+type CartItem implements Node {
+  "The cart item ID"
+  _id: ID!
+
+  "The date and time at which this item was added to the associated cart. If an item is added, removed, and then added again, this will reflect the most recent addition."
+  addedAt: DateTime! # Should this be "createdAt" ?
+
+  "The date and time at which the cart item was created. If an item is added, removed, and then added again, the original item is destroyed and this field will reflect the time it was created for the most recent addition."
+  createdAt: DateTime!
+
+  "Arbitrary additional metadata about this cart item."
+  metafields: [Metafield]
+
+  "Packing information such as item weight, height, length, and depth."
+  parcel: ShippingParcel
```

Comment:
Continuing the conversation in the previous diff... It's fine if this is one of the things we want to denormalize off the product. I noticed that our ShippingParcel schema currently has no units. Instead, we do this in the code:

```js
const unitOfMeasure = (shop && shop.baseUOM) || "kg";
const unitOfLength = (shop && shop.baseUOL) || "cm";
```

By contrast, Shippo has the units right in the parcel definition. Should we update the `ShippingParcel` schema to add `distanceUnit` and `massUnit` (same names Shippo uses)?